istio/scripts/setup-final.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
+    
+sudo snap install kubectl --classic
+
+curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-linux-arm64
```

Comment:
I went to test this in the interest of thoroughness, but was not able to because I'm on a boring ole AMD64 arch machine, and this makes the sample app arm64 bound.

It looks like in the docker setup you're doing some checking for the arch before adding the apt repository. Similar logic here would be appreciated. :)

Reply:
Actually, it appears there is a custom multipass VM that will start with minikube installed and in place.

https://multipass.run/docs/get-started-with-multipass-linux#heading--create-a-customised-instance

Reply:
@rgeyer Hello. Getting back to this after coming back from our work retreat. So there were issues with using the multipass VM with minikube baked in (I think maybe it was amd64 only?).

I will modify this so it should hopefully work on both arm64/amd64 and then let you know.

Reply:
done